cibyl/config.py

```diff
@@ -157,10 +156,8 @@ def _verify_by_schema(self):
             validators = Draft7ValidatorFactory()
             validator = validators.from_file(self._schema)
 
-            try:
-                validator.validate(self.data)
-            except ValidationError as ex:
-                raise SchemaError(error=ex.message)
+            if not validator.is_valid(self.data):
+                raise SchemaError(error="Invalid configuration file.")
```

Comment:
I am retiring '.validate' in favor of '.is_valid'. They do the same thing in the end.